dart/example/monster_my_game.sample_generated.dart

```diff
@@ -0,0 +1,456 @@
+  @override
+  Vec3 get pos {
+    _pos ??= 
+        Vec3.reader.vTableGet(_bc, _bcOffset, 4, null);
+    return _pos;
+  }
+
+  int _mana;
+  @override
+  int get mana {
+    _mana ??= 
+        const fb.Int16Reader().vTableGet(_bc, _bcOffset, 6, 150);
+    return _mana;
+  }
+
+  int _hp;
+  @override
+  int get hp {
+    _hp ??= 
+        const fb.Int16Reader().vTableGet(_bc, _bcOffset, 8, 100);
+    return _hp;
+  }
+
+  String _name;
+  @override
+  String get name {
+    _name ??= 
+        const fb.StringReader().vTableGet(_bc, _bcOffset, 10, null);
+    return _name;
+  }
+
+  List<int> _inventory;
+  @override
+  List<int> get inventory {
+    _inventory ??= 
+        const fb.ListReader<int>(const fb.Uint8Reader()).vTableGet(_bc, _bcOffset, 14, null);
+    return _inventory;
+  }
+
+  Color _color;
+  @override
+  Color get color {
+    _color ??= new Color.fromValue(
+        const fb.Int8Reader().vTableGet(_bc, _bcOffset, 16, 2));
+    return _color;
+  }
+
+  List<Weapon> _weapons;
+  @override
+  List<Weapon> get weapons {
+    _weapons ??= 
+        const fb.ListReader<Weapon>(Weapon.reader).vTableGet(_bc, _bcOffset, 18, null);
+    return _weapons;
+  }
+
+  EquipmentTypeId _equippedType;
+  @override
+  EquipmentTypeId get equippedType {
+    _equippedType ??= new EquipmentTypeId.fromValue(
+        const fb.Uint8Reader().vTableGet(_bc, _bcOffset, 20, null));
+    return _equippedType;
+  }
+
+  dynamic _equipped;
+  @override
+  dynamic get equipped {
+    switch (equippedType?.value) {
+      case 1: return Weapon.reader.vTableGet(_bc, _bcOffset, 22, null);
+      default: return null;
+    }
+  }
+
+  @override
+  String toString() {
+    return 'Monster{pos: $pos, mana: $mana, hp: $hp, name: $name, inventory: $inventory, color: $color, weapons: $weapons, equippedType: $equippedType, equipped: $equipped}';
+  }
+}
+
+class _MonsterReader extends fb.TableReader<_MonsterImpl> {
+  const _MonsterReader();
+
+  @override
+  _MonsterImpl createObject(fb.BufferContext bc, int offset) => 
+    new _MonsterImpl(bc, offset);
+}
+
+class MonsterBuilder extends fb.BuilderHelper {
+  final Vec3Builder _pos;
+  final int _mana;
+  final int _hp;
+  final String _name;
+  final List<int> _inventory;
+  final Color _color;
+  final List<WeaponBuilder> _weapons;
+  final EquipmentTypeId _equippedType;
+  final dynamic _equipped;
+
+  MonsterBuilder({
+    Vec3Builder pos,
+    int mana,
+    int hp,
+    String name,
+    List<int> inventory,
+    Color color,
+    List<WeaponBuilder> weapons,
+    EquipmentTypeId equippedType,
+    dynamic equipped,
+  })
+      : _pos = pos,
+        _mana = mana,
+        _hp = hp,
+        _name = name,
+        _inventory = inventory,
+        _color = color,
+        _weapons = weapons,
+        _equippedType = equippedType,
+        _equipped = equipped;
+
+  /// Finish building, and store into the [fbBuilder].
+  @override
+  int finish(
+    fb.Builder fbBuilder, { bool internStrings = false }) {
```

Comment:
The internstrings bool should really be in the builder object that contains the string map?

Reply:
This flag gets passed through to that Builder.  This way, particular tables/objects can choose to intern strings or not.

Reply:
It be fine to determine that globally, I think.

Reply:
I'd like to avoid global variables, and I like the idea of being able to intern strings for some objects but not others...  This allows for that.

Reply:
Got it now, this should be an easy change.